modules/@apostrophecms/ui/ui/apos/composables/AposTheme.js

```diff
@@ -1,11 +1,12 @@
 import { computed } from 'vue';
 
-export function useAposTheme () {
-  const themeClass = computed(() => {
-    const classes = [];
-    classes.push(`apos-theme--primary-${window.apos.ui.theme.primary}`);
-    return classes;
-  });
-
+export function useAposTheme() {
+  const themeClass = computed(_themeClass);
   return { themeClass };
 };
+
+export function _themeClass() {
+  const classes = [];
+  classes.push(`apos-theme--primary-${window.apos.ui.theme.primary}`);
+  return classes;
+}
```

Comment:
Not related to this ticket, but I used the same sharing code logic for ApostTheme and AposThemeMixin.